plugin.video.vrt.nu/resources/lib/vrtplayer/vrtapihelper.py

```diff
@@ -31,11 +31,13 @@ def get_tvshow_items(self, path):
         tvshows = requests.get(api_url, proxies=self._proxies).json()
         tvshow_items = []
         for tvshow in tvshows:
-            metadata_creator = metadatacreator.MetadataCreator()
-            metadata_creator.mediatype = 'tvshow'
-            metadata_creator.tvshowtitle = tvshow.get('title')
-            metadata_creator.plot = statichelper.unescape(tvshow.get('description'))
-            metadata_creator.brands = tvshow.get('brands')
+            metadata = metadatacreator.MetadataCreator()
+            metadata.mediatype = 'tvshow'
+            metadata.tvshowtitle = tvshow.get('title')
+            metadata.plot = statichelper.unescape(tvshow.get('description'))
+            metadata.plotoutline = metadata.plot
+            metadata.brands = tvshow.get('brands')
+            metadata.score = tvshow.get('score')  # NOTE: The score here is always between 4.8 and 5.0
```

Comment:
I think score means the relevance of the search query here, for a different query, you get a different score:
https://vrtnu-api.vrt.be/search?i=video&facets[url]=//www.vrt.be/vrtnu/a-z/inside-europe--ten-years-in-turmoil/2018/inside-europe--ten-years-in-turmoil-s2018a2/

It's not related with a user rating or something else.

Reply:
For programs they are quite low (between 4.8 and 5.1) but for episodes they seem to indicate something else, which I think is popularity or personal preference based on data mining. All the programs I like to watch score very high (sometimes higher than 10), but cycling tours score even higher (which I do not watch, except for Tour de France in the background).

So I don't think they are related just to search relevance, but rather popularity.

Reply:
Strange, Gent-Wevelgem has a score of 13.495115 and 44.903103:

https://vrtnu-api.vrt.be/search?i=video&facets[url]=//www.vrt.be/vrtnu/a-z/sporza--wielrennen/2019/sporza--wielrennen-d20190331-gent-wevelgem/
https://vrtnu-api.vrt.be/search?i=video&q=pbs-pub-8ff1a6c9-bdd1-4404-960f-a94a80b0c7b3

Reply:
Indeed, TBH I think this means that a lot of cycling fans use VRT.NU (e.g. when they are actually live at the event). So the popularity numbers are inflated for cycling because of the sheer number of users who watch it through VRT.NU, whereas most other programs are watch most through broadcasting.

So maybe these outliers are exceptions to the data mining algorithms (in case these are somehow personal recommendations based on profile/viewing habit) or simply a general popularity rating with very few or none personal information (is my profile even used for the listings, I don't think so).

Reply:
I mean: one single video item has two different scores dependent on the type of api query.

https://www.elastic.co/guide/en/elasticsearch/guide/current/scoring-theory.html

Reply:
But we didn't provide any keywords, so what would it be scoring against?

Reply:
BTW I agree that in case this offers no real value, we shouldn't be adding it as score. But I wanted to provoke this kind of discussion by publishing this Work-In-Progress, so thanks in advance.